tests/src/toCKB_typescript/tests/liquidation_signertimeout.rs

```diff
@@ -1 +1,182 @@
+use super::{Byte32, Script, ToCKBCellData};
+use crate::*;
+use ckb_testtool::{builtin::ALWAYS_SUCCESS, context::Context};
+use ckb_tool::ckb_types::{
+    bytes::Bytes,
+    core::{TransactionBuilder, TransactionView},
+    packed::*,
+    prelude::*,
+};
+use ckb_tool::{ckb_error::assert_error_eq, ckb_script::ScriptError};
+use molecule::prelude::*;
+use int_enum::IntEnum;
 
+
+const MAX_CYCLES: u64 = 10_000_000;
+const PLEDGE_INVALID: i8 = 8;
+const LOT_SIZE_INVALID: i8 = 7;
+const TX_INVALID: i8 = 6;
+const ENCODING: i8 = 4;
+const INVARIANT_DATA_MUTATED: i8 = 10;
+
+#[repr(u8)]
+#[derive(Clone, Copy, IntEnum, PartialEq)]
+pub enum ToCKBStatus {
+    Initial = 1,
+    Bonded = 2,
+    Warranty = 3,
+    Redeeming = 4,
+    SignerTimeout = 5,
+    Undercollateral = 6,
+    FaultyWhenWarranty = 7,
+    FaultyWhenRedeeming = 8,
+}
```

Comment:
remove it, import from utils/types instead

Reply:
fixed